README.md

```diff
@@ -2,9 +2,9 @@
 
 A `bfloat16` implementation for projects of [BioVault](https://github.com/biovault) (Biomedical Visual Analytics Unit LUMC - TU Delft)
 
-Originally based upon `dnnl::impl::bfloat16_t` from the [Deep Neural Network Library (DNNL)](https://github.com/intel/mkl-dnn) of Intel Corporation:
-* https://github.com/intel/mkl-dnn/blob/v1.2/src/cpu/bfloat16.cpp
-* https://github.com/intel/mkl-dnn/blob/v1.2/src/common/bfloat16.hpp
+Originally based upon `dnnl::impl::bfloat16_t` from the [oneAPI Deep Neural Network Library (oneDNN)](https://github.com/oneapi-src/oneDNN) of Intel Corporation:
```

Comment:
I would not say Originally here because it was originally based on the mkl-dnn library